test/shmemx/Makefile.am

```diff
@@ -31,23 +25,27 @@ check_PROGRAMS += \
 	shmem_ct
 endif
 
-if SHMEMX_TESTS
-
 check_PROGRAMS += \
 	perf_counter \
 	fadd_nbi \
 	atomic_nbi \
 	put_signal \
-	put_signal_nbi
+	put_signal_nbi \
+	shmemx_wait_until_all \
+	shmemx_wait_until_any \
+	shmemx_wait_until_some \
+	shmemx_test_all \
+	shmemx_test_any \
+	shmemx_test_some \
+	c11_test_shmemx_wait_until \
+	c11_test_shmemx_test
 
 if HAVE_PTHREADS
 check_PROGRAMS += \
 	gettid_register
 endif
-endif SHMEMX_TESTS
 
 if HAVE_CXX
-if SHMEMX_TESTS
```

Comment:
C++ generic bindings are an extension and should have `SHMEMX_TESTS` around them.

Reply:
This does have `SHMEMX_TESTS` around it - if I'm not mistaken...  Perhaps the git diff is misleading?